app/lib/file_parse_service.rb

```diff
@@ -97,6 +97,24 @@ def self.run_parse_job(study_file, study, user, reparse: false, persist_on_fail:
         else
           Rails.logger.info "Aborting parse of #{@study_file.name} as #{@study_file.file_type} in study #{@study.name}; not applicable"
         end
+      when 'AnnData'
+        # gate ingest of AnnData using the feature flag 'ingest_anndata_file'
+        do_anndata_file_ingest = FeatureFlaggable.feature_flags_for_instances(user, study)['ingest_anndata_file']
+        if do_anndata_file_ingest == false
```

Comment:
Non-blocking:  I think it might be more readable to invert this and have the `true` clause first.  You could write it as:
```
if do_anndata_file_ingest
  ... ingest code
else
  ... log message for aborting call
end
```